### PR TITLE
TIP-613: Fix boolean form on PEF association datagrid

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

This PR fixes boolean field on the PEF association datagrid.

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | OK
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/cell/boolean-label-cell.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ define(['oro/datagrid/string-cell', 'oro/translator'],
                     return this;
                 }
 
-                var status = 'true' === value ? 'success' : 'important';
-                var label = 'true' === value ? 'Yes' : 'No';
+                var status = (true === value || '1' === value) ? 'success' : 'important';
```
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/cell/boolean-label-cell.js]: I don't remember why there was simple quote before around `true`, but I think you break the association products datagrid if you remove them

Comment (damien-carcel) [reply src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/cell/boolean-label-cell.js]: Nope, in my case it is what fixes it, the YES/NO field is now working.

Comment (damien-carcel) [reply src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/cell/boolean-label-cell.js]: But maybe there is another case, and we need the three of them: true, 'true' and '1'
#viandedatagrid

**Review by skeleton — APPROVED** [review]

**Review by SamirBoulil — APPROVED** [review]
Nice one ^^

**Pull request merged**